src/main/java/org/example/cli/Employee.java

```diff
@@ -0,0 +1,86 @@
+package org.example.cli;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Employee {
+    // Variable declaration
+
+    private int id;
+    private String first_name;
+    private String last_name;
+    private double salary;
+    private String bank_account_number;
+    private String ni_number;
+
+    // Constructor to create new employee with all required details
+    @JsonCreator
+    public Employee(
+            @JsonProperty("id") int id,
+            @JsonProperty("first_name") String first_name,
+            @JsonProperty("last_name") String last_name,
+            @JsonProperty("salary") double salary,
+            @JsonProperty("bank_account_number") String bank_account_number,
+            @JsonProperty("ni_number") String ni_number) {
```

Comment:
The variables aren't the normal naming convention for Java variables

Reply:
Variable names have now been updated :)